src/anomalib/cli/cli.py

```diff
@@ -296,6 +297,9 @@ def instantiate_classes(self) -> None:
             # the minor change here is that engine is instantiated instead of trainer
             self.config_init = self.parser.instantiate_classes(self.config)
             self.datamodule = self._get(self.config_init, "data")
+            if isinstance(self.datamodule, Dataset):
+                param = {f"{self.config.subcommand}_dataset": self.datamodule}
```

Comment:
a bit super minor and opinionated, but was wondering whether `kwargs` would be a better naming here or not?